govcd/ip_space_allocation.go

```diff
@@ -279,3 +281,30 @@ func getAllIpSpaceAllocations(client *Client, ipSpaceId string, org *Org, queryP
 
 	return results, nil
 }
+
+// GetAllIpSpaceFloatingIpSuggestions suggests IP addresses to use for networking services on Edge
+// Gateway or Provider Gateway. 'gatewayId' is mandatory. Based on the specified Gateway, VCD
+// will query all the applicable IP Spaces and suggest some IP addresses which can be utilized to
+// configure the network services on the Gateway. IP Space IP addresses which are are allocated but
+// not currently used for any network services are considered. Results can also be filtered by IPV4
```

Comment:
A typo. I'm also thinking how to make this description feel less "floaty" (i.e. make it more determined) :)

Something like this:

```suggestion
// configure the network services on the Gateway. Allocated IP Space's IP addresses, but
// not currently used for any network services are returned. Results can also be filtered by IPV4
```

Reply:
I have set it to be such. This comes from API docs, but indeed might sound confusing.